aiorussound/rio.py

```diff
@@ -232,6 +235,93 @@ async def init_sources(self) -> None:
             except CommandError:
                 break
 
+    async def enumerate_zone_favorites(self, zone: Zone) -> list[RussoundFavorite]:
+        """Return a list of RussoundFavorite for this zone."""
+        favorites = []
+
+        for favorite_id in range(1, 2):
```

Comment:
I'm seeing in the API documentation that there can be upto 4 zone favorites (or only 2 if it's an XStream series). We should probably add some logic to check the controller type to determine this value to prevent unnecessary network calls.

Reply:
I was using an old rev.  Updated to account for difference between controller types